alpha/lib/model/DeliveryProfileVod.php

```diff
@@ -75,21 +75,21 @@ protected function doGetFlavorAssetUrl(flavorAsset $flavorAsset)
 	 * @param thumbAsset $thumbAsset
 	 * @return string
 	 */
-	protected function doGetThumbnailAssetUrl(thumbAsset $thumbAsset)
+	protected function doGetThumbnailAssetUrl(thumbAsset $thumbAsset, $addExtension = false)
 	{
 		$thumbAssetId = $thumbAsset->getId();
 		$partnerId = $thumbAsset->getPartnerId();
-		$url = "/api_v3/service/thumbAsset/action/serve/partnerId/$partnerId/thumbAssetId/$thumbAssetId";
+		$url = "/api_v3/service/thumbAsset/action/serve/partnerId/$partnerId/thumbAssetId/$thumbAssetId" . ($addExtension ?  "/$thumbAssetId." . $thumbAsset->getFileExt() : "");
```

Comment:
@assafb This is intentional. The URL provided here is for the thumbAsset->serve API. The URL will look like this:

http://{hostname}/api_v3/service/thumbAsset/action/serve/partnerId/$partnerId/thumbAssetId/$thumbAssetId/$thumbAssetId.ext